doc/luasnip.txt

```diff
@@ -152,14 +153,15 @@ The first parameter of `f` is the function. Its parameters are
 
 The second parameter is a table of indizes of jumpable nodes whose text is
 passed to the function. The table may be empty, in this case the function is
-evaluated once upon snippet-expansion.
+evaluated once upon snippet-expansion. If the table only has a single node, it
+can be passed directly without wrapping it in a table.
 
 Examples:
 	Use captures from the regex-trigger using a functionNode:
 >
 	s({trig = "b(%d)", regTrig = true, wordTrig = true}, {
 		f(function(args) return
-			{"Captured Text: " .. args[1].captures[1] .. "."} end, {}),
+			"Captured Text: " .. args[1].captures[1] .. "." end, {}),
```

Comment:
I think the example in line 138 doesn't need { } either, could you add that?

Reply:
Also, I didn't add anything about the functions return-value, could you add that it can be either string or table?